src/test/runTest.ts

```diff
@@ -0,0 +1,54 @@
+import * as path from "path";
+import * as fs from "fs";
+import * as tmp from "tmp";
+
+import { runTests } from "vscode-test";
+import { assert } from "console";
+
+const PROJECT_ROOT = path.join(__dirname, "..", "..");
+const DATA_ROOT = path.join(PROJECT_ROOT, "src", "test", "data");
+
+async function main() {
+    // The folder containing the Extension Manifest package.json
+    // Passed to `--extensionDevelopmentPath`
+    const extensionDevelopmentPath = path.resolve(__dirname, "../../");
+
+    // The path to the extension test runner script
+    // Passed to --extensionTestsPath
+    const extensionTestsPath = path.resolve(__dirname, "./index");
+
+    // Download VS Code, unzip it and run the integration test
+    console.info("Reading list of settings...");
+    const settings_list = fs.readdirSync(path.join(DATA_ROOT, "settings"));
+    assert(settings_list.length > 0, "There are no settings to test");
+
+    for (const settings_file of settings_list) {
```

Comment:
@ArquintL you might be interested in this if you need to run the full test suite on multiple settings files.